Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -234,7 +239,7 @@ import BraintreeDataCollector
         request: BTPayPalRequest,
         completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
     ) {
-        self.apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeStarted)
+        apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeStarted, isVaultRequest: isVaultRequest)
```

Comment:
I think we also want to send this anywhere we are calling `sendAnalyticsEvent` in this flow?

Reply:
When you say "this flow", do you mean the BT PayPal Client flow?

Reply:
I reread the ticket and assume yes, so I've made the changes in `BTPayPalClient` and have done the same in `BTVenmoClient`